src/MBTool.cpp

```diff
@@ -11,6 +11,14 @@ const char geom_categories[][CATEGORY_TAG_SIZE] = {"Vertex\0",
 						   "Group\0"};
 
 
+
+static inline void check_moab_rval(moab::ErrorCode ret, const char * what) {
+  if (ret == moab::MB_SUCCESS) {
+    return;
+  }
+  throw mberror(ret, what);
+}
```

Comment:
What is the reason for doing it that way round, with the early return, rather than the shorter version with no early return?
```suggestion
static inline void check_moab_rval(moab::ErrorCode ret, const char * what) {
  if (ret != moab::MB_SUCCESS) {
    throw mberror(ret, what);
  }
}
```